api/v1alpha1/event.go

```diff
@@ -40,5 +40,6 @@ const (
 	EventActorEmailPrefix          = "email:"
 	EventActorSubjectPrefix        = "subject:"
 	EventActorKubernetesUserPrefix = "kubernetes:"
+	EventActorOidcUsernamePrefix   = "oidc-username:"
 	EventActorUnknown              = "unknown actor"
 )
```

Comment:
I think that if, historically, the prefix has been `email:` when we extracted actor information from an `email` claim and `subject:` when we've extracted it from the `sub` claim (ya... there's a slight mismatch there), the right thing to do, in general, would be to use whatever claim the operator specified as the prefix.

I would probably just remove this line altogether and then refer to [this comment]() where you'll see we no longer need this constant.

```suggestion
	EventActorEmailPrefix          = "email:"
	EventActorSubjectPrefix        = "subject:"
	EventActorKubernetesUserPrefix = "kubernetes:"
	EventActorUnknown              = "unknown actor"
)

var EventActorOIDCClaimPrefix = os.GetEnv("OIDC_USERNAME_CLAIM", "email") + ":"
```